src/tests/test_example_diffs.rs

```diff
@@ -837,38 +837,115 @@ src/align.rs
-        _do_test_output_is_in_one_to_one_correspondence_with_input(&[
-            "--color-only",
-            "true",
-            "--hunk-header-style",
-            "normal",
-            "--line-numbers",
-        ]);
-        _do_test_output_is_in_one_to_one_correspondence_with_input(&[
-            "--color-only",
-            "true",
-            "--file-style",
-            "blue",
-            "--commit-style",
-            "omit",
-            "--hunk-header-style",
-            "omit",
-            "--hunk-header-decoration-style",
-            "omit",
-        ]);
-        _do_test_output_is_in_one_to_one_correspondence_with_input(&[
-            "--color-only",
-            "true",
-            "--file-style",
-            "blue",
-            "--commit-style",
-            "red",
-            "--hunk-header-style",
-            "syntax",
-            "--hunk-header-decoration-style",
-            "box",
-        ]);
+        let user_suppliable_configs: &[&[&str]] = &[
+            &["--color-only", "--light", "true"],
+            &["--color-only", "--dark", "true"],
+            &["--color-only", "--line-numbers", "true"],
+            &["--color-only", "--side-by-side", "true"],
+            &["--color-only", "--diff-highlight", "true"],
+            &["--color-only", "--diff-so-fancy", "true"],
+            &["--color-only", "--navigate", "true"],
+            &["--color-only", "--hyperlinks", "true"],
+            &["--color-only", "--keep-plus-minus-markers", "true"],
+            &["--color-only", "--raw", "true"],
+            &["--color-only", "--syntax-theme", "Monokai Extended"],
+            &["--color-only", "--minus-style", "syntax bold auto"],
+            &["--color-only", "--zero-style", "red strike"],
+            &["--color-only", "--plus-style", "red blink"],
+            &["--color-only", "--minus-emph-style", "red dim"],
+            &["--color-only", "--minus-non-emph-style", "red hidden"],
+            &["--color-only", "--plus-emph-style", "red reverse"],
+            &["--color-only", "--plus-non-emph-style", "red bold ul"],
+            &["--color-only", "--commit-style", "red bold ul"],
+            &[
+                "--color-only",
+                "--commit-decoration-style",
+                "bold yellow box ul",
+            ],
+            &["--color-only", "--file-style", "red bold ul"],
+            &[
+                "--color-only",
+                "--file-decoration-style",
+                "bold yellow box ul",
+            ],
+            &["--color-only", "--hunk-header-style", "red bold ul"],
+            &[
+                "--color-only",
+                "--hunk-header-decoration-style",
+                "bold yellow box ul",
+            ],
+            &[
+                "--color-only",
+                "--line-numbers",
+                "true",
+                "--line-numbers-minus-style",
+                "#444444",
+            ],
```

Comment:
Apparently, these `--line-numbers-***` is not necessary because `line-numbers` would be disabled. However, I can't make sure if the option's behavior will be changed, so I added anyway.

It's just my opinion.